supplements/adventurers-league/dnd-celebration-2020.xml

```diff
@@ -0,0 +1,282 @@
+			<set name="alignment">unaligned</set>
+			<set name="challenge">0</set>
+			<set name="traits">ID_WOTC_DNDC_COMPANION_TRAIT_SNOWY_OWLBEAR_CUB_KEEN_SMELL</set>
+			<set name="actions">ID_WOTC_DNDC_COMPANION_ACTION_SNOWY_OWLBEAR_CUB_CLAWS</set>
+		</setters>
+		<rules>
+			<stat name="companion:ac" value="12" />
+			<stat name="companion:hp:max" value="2" bonus="base" />
+			<stat name="companion:speed" value="40" bonus="base" />
+			<stat name="companion:speed:climb" value="30" bonus="base" />
+			<stat name="companion:perception:proficiency" value="companion:proficiency" bonus="base" />
+			<stat name="companion:stealth:proficiency" value="companion:proficiency" bonus="base" />
+		</rules>
+	</element>
+	
+	<element name="Keen Smell" type="Companion Trait" source="D&amp;D Celebration 2020" id="ID_WOTC_DNDC_COMPANION_TRAIT_SNOWY_OWLBEAR_CUB_KEEN_SMELL">
+		<description>
+			<p>The snowy owlbear cub has advantage on Wisdom (Perception) checks that rely on smell.</p>
+		</description>
+		<sheet>
+			<description>The snowy owlbear cub has advantage on Wisdom (Perception) checks that rely on smell.</description>
+		</sheet>
+	</element>
+	
+	<element name="Claws" type="Companion Action" source="D&amp;D Celebration 2020" id="ID_WOTC_DNDC_COMPANION_ACTION_SNOWY_OWLBEAR_CUB_CLAWS">
+		<description>
+			<p>Melee Weapon Attack: +0 to hit, reach 5 ft., one target. Hit: 1 slashing damage.</p>
+		</description>
+		<sheet>
+			<description>Melee Weapon Attack: +0 to hit, reach 5 ft., one target. Hit: 1 slashing damage.</description>
+		</sheet>
+	</element>
+	<!--/Snowy Owlbear Cub-->
+
+	<!--Items-->
+	<element name="Inspired Cocoa Mug" type="Magic Item" source="D&amp;D Celebration 2020" id="ID_WOTC_DNDC_MAGIC_ITEM_INSPIRED_COCOA_MUG">
+		<description>
+			<p>This mug can magically fill with delicious hot cocoa during a short rest. Drinking the cocoa allows one creature to regain an extra 1d6 hit points if they spend Hit Dice to regain hit points.</p>
+		</description>
+		<setters>
+			<set name="keywords"></set>
+			<set name="category">Wondrous Items</set>
+			<set name="cost" currency="gp">0</set>
+			<set name="weight" lb="0">0 lb.</set>
+			<set name="type">Wondrous Item</set>
+			<set name="rarity">Unknown</set>
+		</setters>
+	</element>
+
+	<element name="Netherese Ghost Stories" type="Magic Item" source="D&amp;D Celebration 2020" id="ID_WOTC_DNDC_MAGIC_ITEM_NETHERESE_GHOST_STORIES">
+		<description>
+			<p>Reading from this Netherese book of horrifying ghost stories for 10 minutes grants advantage on a single related History check and gives the reader nightmares. You must succeed on a DC 13 Wisdom saving throw to successfully complete your next long rest.</p>
+		</description>
+		<setters>
+			<set name="keywords">saving throw</set>
+			<set name="category">Wondrous Items</set>
+			<set name="cost" currency="gp">0</set>
+			<set name="weight" lb="0">0 lb.</set>
+			<set name="type">Wondrous Item</set>
+			<set name="rarity">Unknown</set>
```

Comment:
It's preferable to not create new rarities, if it is unknown, it's better to just not use the setter at all, as it will leave that bit out of description completely.
```suggestion
```